### PR TITLE
MTL-1681 Add a notice for interface name changes

### PR DESCRIPTION
**Pull request by rustydb**
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

The PITs NICs may change names if Mellanox cards are used on the system. This notice directs users to validate their inputs to CSI prior to generating their configuration payload.

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

N/A

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [MTL-1681](https://jira-pro.its.hpecorp.net:8443/browse/MTL-1681)
* Change will also be needed in `release/1.2.5` and `release/1.2`
* Future work required by [CASMINST-4425](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4425)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `jolt1` and `fanta`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

Both systems were impacted by CASMINST-4425, the users installing those systems used these directions to proceed.

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

No.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: install/prepare_configuration_payload.md [hunk 1]
```diff
@@ -210,6 +210,21 @@ The process to reinstall must have the configuration payload files available.
...
+      >
+      >     ```bash
+      >     <lan> <vid>:<did>
+      >     pit# lid
+      >     p238p1 8086:1521
+      >     p238p2 8086:1521
+      >     p238p3 15B3:1013
+      >     p238p4 15B3:1013
+      >
+      > 1. Update `system_config.yaml`'s `install-ncn-bond-members` valye with the interface names cross-referenced from the `lid` output.
```
Comment (gbaker-hpe) [inline install/prepare_configuration_payload.md:225]: small typo

File: install/prepare_configuration_payload.md [hunk 2]
```diff
@@ -210,6 +210,21 @@ The process to reinstall must have the configuration payload files available.
...
+      > 1. Compare the output of `lid` with the background document here for PCIe devices. Cross-reference the Vendor ID with the IDs in the table in [Vendor and Bus ID Identification](../background/ncn_networking.md#vendor-and-bus-id-identification).
+      >
+      >     ```bash
+      >     <lan> <vid>:<did>
+      >     pit# lid
+      >     p238p1 8086:1521
+      >     p238p2 8086:1521
+      >     p238p3 15B3:1013
+      >     p238p4 15B3:1013
+      >
```
Comment (mharding-hpe) [inline install/prepare_configuration_payload.md:224]: It looks like you are missing the backticks here.
```suggestion
      >     ```
      >
```

File: install/prepare_configuration_payload.md (hunk 1 above)
Comment (mharding-hpe) [inline install/prepare_configuration_payload.md:225]:
```suggestion
      > * Update `system_config.yaml`'s `install-ncn-bond-members` value with the interface names cross-referenced from the `lid` output.
```

File: install/prepare_configuration_payload.md [hunk 3]
```diff
@@ -210,6 +210,21 @@ The process to reinstall must have the configuration payload files available.
...
+      > 1. Update `system_config.yaml`'s `install-ncn-bond-members` valye with the interface names cross-referenced from the `lid` output.
+      > 1. _Alternatively_ run `csi config init --install-ncn-bond-members` with the discovered interface names.
```
Comment (mharding-hpe) [inline install/prepare_configuration_payload.md:226]:
```suggestion
      > * _Alternatively_, run `csi config init --install-ncn-bond-members` with the discovered interface names.
```

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**